habitat-lab/habitat/config/read_write.py

```diff
@@ -1,15 +1,18 @@
 from contextlib import contextmanager
-from typing import Generator, Union
+from typing import TYPE_CHECKING, Generator, Union
 
 from omegaconf import OmegaConf
 from omegaconf.base import Node
 
-from habitat.config.default import Config
+if TYPE_CHECKING:
+    from omegaconf import DictConfig
 
 
 # TODO : Delete this when migration to hydra is complete
 @contextmanager
-def read_write(config: Union[Node, Config]) -> Generator[Node, None, None]:
+def read_write(
+    config: Union[Node, "DictConfig"]
+) -> Generator[Node, None, None]:
     if isinstance(config, Node):
```

Comment:
This can now be removed. There is no need to check here. We should remove anything below line 26 as this was for compatibility with YACS.